src/Postnl.php

```diff
@@ -238,15 +244,18 @@ public function generateBarcodeByDestination(
         $customerNumber = null,
         $serie = null
     ) {
+        $eps = false;
```

Comment:
No, than global shipments (worldwide) would have $eps (EU) true passed as well. Although the same label is generated, it is not correct.